src/query/functions/tests/it/scalars/testdata/string.txt

```diff
@@ -499,28 +395,28 @@ output         : ''
 ast            : reverse('你好')
 raw expr       : reverse('你好')
 checked expr   : reverse<String>("你好")
-optimized expr : 0xbda5e5a0bde4
+optimized expr : BDA5E5A0BDE4
```

Comment:
seems the output is invalid

Reply:
let's leave the string function update to the next pr.